netconf_client/ncclient.py

```diff
@@ -24,6 +24,14 @@
     delete_config,
 )
 
+import six
+if six.PY3:
+    from time import monotonic
+    _current_timestamp = monotonic
```

Comment:
I think it may be better to call this `_get_current_timestamp`, so that it doesn't have the same name as the `current_timestamp` variable in the loop below.

Reply:
I have renamed the fuction reference as you suggested.
The bugfix version number is updated.
I had to change a few things because of Pylama.